pages/data/index.vue

```diff
@@ -479,6 +526,28 @@ export default {
         this.filters
       )
       this.filters = filters
+      this.setTagsQuery()
+    },
+
+    /**
+     * Set the tags query parameter in the router
+     */
+    setTagsQuery: function() {
+      const filterVals = this.activeFilters.map(filter => {
+        return filter.key
+      })
+
+      const queryParamTags = pathOr('', ['query', 'tags'], this.$route)
+      if (equals(filterVals, queryParamTags.split(','))) {
+        return
+      }
+
+      const tags = { tags: filterVals.join(',') }
+
+      const query = mergeLeft(tags, this.$route.query)
```

Comment:
`const query = { ...this.$route.query, ...tags }`

Reply:
I'm pretty sure `mergeLeft` behaves the same way as the spread operator after reading the doc again.  I could be wrong...